content/integrations/awsbilling.md

```diff
@@ -30,4 +30,18 @@ AWS Budget metrics can only be collected from the AWS master account.
 
 {{< get-metrics-from-git >}}
 
-Each of the metrics retrieved from AWS will be assigned the same tags that appear in the AWS console, including but not limited to host name, security-groups, and more.
\ No newline at end of file
+Each of the metrics retrieved from AWS will be assigned the same tags that appear in the AWS console, including but not limited to host name, security-groups, and more.
+
+## Troubleshooting
+
+### No metric are reporting from my AWS billing integration
+Here is a check list that you can apply to troubleshoot your integration:
+
+1. Check that your IAM policy has `budgets:ViewBudget`
+2. Check if billing metrics is enabled in your payer account
```

Comment:
Ensure billing metrics are enabled in your payer account

Reply:
👍